lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -127,6 +141,51 @@ def build_credit_request(amount, payment, options)
         end.compact
       end
 
+      def create_customer(payment, options)
+        { buyerInformation: {}, clientReferenceInformation: {}, merchantDefinedInformation: [] }.tap do |post|
+          post[:buyerInformation][:merchantCustomerId] = options[:customer_id]
+          post[:buyerInformation][:email] = options[:email].presence || 'null@cybersource.com'
+          add_code(post, options)
+        end.compact
+      end
+
+      def create_instrument_identifier(payment, options)
+        instrument_identifier = {
+          card: {
+            number: payment.number
+          }
+        }
+        commit('instrumentidentifiers', instrument_identifier)
+      end
+
+      def create_payment_instrument(payment, instrument_identifier, options)
+        post = {
+          card: {
+            expirationMonth: payment.month.to_s,
+            expirationYear: payment.year.to_s,
+            type: payment.brand
+          },
+          instrumentIdentifier: {
+            id: instrument_identifier.params['id']
+          }
+        }
+        add_address(post, payment, options[:billing_address], options, :billTo, nil)
+        commit('paymentinstruments', post)
+      end
+
+      def create_customer_payment_instrument(payment, options, customer_id, instrument_identifier)
+        post = {}
+        post[:deafult] = 'true'
+        post[:card] = {}
+        post[:card][:type] = CREDIT_CARD_CODES[payment.brand.to_sym]
+        post[:card][:expirationMonth] = payment.month.to_s
```

Comment:
❓ Question:

Is not sensitive data, but is a must to send the data even when you already sent it on the payment_instrument call ?

Reply:
I think I can remove the create_payment_instrument method, considering the response from this method is not used in the subsequent.